packages/vx-stats/src/BoxPlot.tsx

```diff
@@ -16,7 +17,7 @@ type ScaleInput = number;
 
 export type BoxPlotProps = SharedProps & {
   /** Scale for converting ScaleInput values to pixel offsets. */
-  valueScale: GenericScale<ScaleInput>;
+  valueScale: PickD3Scale<ContinuousDomainScaleType, number>;
```

Comment:
should we either use `ScaleInput` here or just remove `ScaleInput` altogether / opt for `number`?